README.md

```diff
@@ -215,6 +219,32 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+### grcov with GitLab CI and codecov.io
+
+```yaml
+codecov:
+  stage:                           coverage
+  variables:
+    CARGO_INCREMENTAL:             0
+    RUSTFLAGS:                     "-Zprofile -Zmir-opt-level=0 -Ccodegen-units=1
+                                      -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
+    # We removed the `-Cinline-threshold=0` flag from the above `RUSTFLAGS` since it was bugged
+    # at the time and lead to inlining of functions that shouldn't be inlined for the coverage
+    # report. (More information here: https://github.com/Kogia-sima/rust-covfix/issues/2)
+  script:
+    # RUSTFLAGS are the cause target cache can't be used here
+    - unset "CARGO_TARGET_DIR" # only needed if it's not default
+    - cargo clean
+    - cargo test --verbose --all-features --no-fail-fast --workspace
+    - cargo build --verbose --all-features --workspace
+    # coverage with branches
+    - grcov ./target -s . -t lcov --llvm --branch --ignore-not-existing --ignore "/*" --ignore "tests/*" -o lcov-w-branch.info
+    - rust-covfix -o lcov-w-branch_correct.info lcov-w-branch.info
```

Comment:
Let's not use rust-covfix, if we want to use it, it should be run automatically by grcov

Reply:
Without it, grcov looses 10-20% accuracy, it's a really good thing. Can it be ran by grcov?